src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -28,7 +28,11 @@ This guide aims to extend the <PlatformLink to="/troubleshooting/">main troubles
 
 ## My `canvas` Elements Aren't Getting Captured
 
-There's currently no support for `canvas`. It's being tracked in this [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/6519). Feel free to 👍 and help us prioritize it.
+Canvas is supported in SDK versions >= `7.94.1`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> for getting started with canvas recordings.
+
+## Replay is slowing down my `canvas`
+
+In order to export images from 3D and WebGL canvases, the integration needs to enable `preserveDrawingBuffer` which can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, then you will need to <PlatformLink to="/session-replay/#canvas-recording-performance">enable manual snapshotting</PlatformLink> and call a `snapshot()` method inside of your re-paint loop.
```

Comment:
```suggestion
The integration needs to enable `preserveDrawingBuffer` to export images from 3D and WebGL canvases. This can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, you'll need to <PlatformLink to="/session-replay/#canvas-recording-performance">enable manual snapshotting</PlatformLink> and call a `snapshot()` method inside of your re-paint loop.
```